event/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "bundler",
+    "moduleResolution": "node",
```

Comment:
Consider changing `moduleResolution` back to the default `bundler`.

From the docs: `node10` (previously called `node`) for Node.js versions older than v10, which only support CommonJS require. You probably won’t need to use node10 in modern code.

https://www.typescriptlang.org/tsconfig#moduleResolution